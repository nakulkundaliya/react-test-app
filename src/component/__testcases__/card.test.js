import React from 'react';
import { shallow } from 'enzyme';
import Card from '../card';
const clickFn = jest.fn();

let wrapped = shallow(<Card repoName="reponame" onClick={clickFn} />);
describe('render card component', () => {
    it('should render the Card component correctly', () => {
        expect(wrapped).toMatchSnapshot();
    });
});