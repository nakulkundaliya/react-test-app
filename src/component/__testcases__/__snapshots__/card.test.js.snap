// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`render card component should render the Card component correctly 1`] = `ShallowWrapper {}`;
